components/sysutils/augeas/Makefile

```diff
@@ -0,0 +1,59 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Jim Klimov
+#
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=	augeas
+COMPONENT_VERSION=	1.4.0
+COMPONENT_REVISION=	1
```

Comment:
I fixed it up and remembered now: the revision was intended for the case if you merge #1818 and #1819 separately - the augtools package metadata is changed then.
If you merge only #1819 as the initial delivery of augtools (and just close #1818), the revision tag is indeed not needed.